src/mono/wasm/Wasm.Build.Tests/BrowserRunner.cs

```diff
@@ -118,6 +120,12 @@ public async Task<IBrowser> SpawnBrowserAsync(
         {
             try
             {
+                // widnows runs are sequential, we cannot terminate when we run in parallel
+                if (s_runsOnWindows)
+                {
+                    TerminateExistingChromeInstances();
```

Comment:
should we also terminate after each test ?
do we believe that chrome process survived on the agent from another run ? this is docker/VM right ?

Reply:
I don't believe after is needed. The problem is with `LaunchAsync`, so the idea was to make sure nothing is running directly before calling that method. And yes, the suspicion was that some leftovers from the previous test was left and that's why new browser launch is time outing. 
This failure is only on CI, I have never managed to reproduce it locally.